packages/cinterop/src/commonMain/kotlin/io/realm/interop/RealmInterop.kt

```diff
@@ -107,9 +107,11 @@ expect object RealmInterop {
     fun <T> realm_list_set(list: NativePointer, index: Long, value: T): T
     fun realm_list_clear(list: NativePointer)
     fun realm_list_erase(list: NativePointer, index: Long)
+    fun realm_list_freeze(liveList: NativePointer, frozenRealm: NativePointer): NativePointer
+    fun realm_list_thaw(frozenList: NativePointer, liveRealm: NativePointer): NativePointer
```

Comment:
Just noticed these are already in master but not in the detination branch. Have you left it outdated intentionally?